pkg/kmodule/kmodule_test.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//go:build darwin || linux
+
+package kmodule
+
+import (
+	"bytes"
+	"testing"
+)
+
+func TestNew(t *testing.T) {
+	l, err := New()
+	if err != nil {
+		t.Errorf("New(): got %v, want nil", err)
```

Comment:
`t.Fatalf` here instead?   If `t.Errorf` is used here then the code on line 21 (`l.List()`) will run, which may lead to unexpected results given that `New()` returned an error.

Reply:
done